app/graphql/mutations/users/signup.rb

```diff
@@ -1,14 +1,13 @@
 module Mutations
   module Users
     class Signup < Mutations::BaseMutation
-      argument :password, String, required: true
-      argument :user_attributes, Types::UserAttributes, required: false
+      argument :user_attributes, Types::Users::UserInput, required: false
 
       field :errors, [String], null: true
       field :token, String, null: true
 
-      def resolve(user_attributes:, password:)
```

Comment:
the reason I separated the `password `from `UserAttribute` is that when users updating user info, users change passwords separately. in user update mutation there will not be password argument and will have `UpdatePassword` mutation.

Reply:
Ah... Make sense! I will revert it